### ISSUE TITLE
Three functions are missing from pprintast

### ISSUE THREAD

**Issue by mattiasdrp** [ocaml-ppx/ppxlib#531]
Thanks for this lib, really useful!

There are three functions that are available in Pprintast that are not in Ppxlib.Pprintast, namely, 

```
val binding : Format.formatter -> Parsetree.value_binding -> unit

val longident : Format.formatter -> Longident.t -> unit

val payload : Format.formatter -> Parsetree.payload -> unit
```

Is there a reason why they're missing or are you planning to add them one day?

**Comment by patricoferris** [comment]
Hi @mattiasdrp,

Thank! I think this is just an oversight. As I understand it, `Pprintast` follows are internal bump of the AST so likely only gets changed whenever we change the AST (e.g. #514). These functions were probably missed in some bump of the AST, it seems they are all available in 4.14. A PR would be very welcome, otherwise I'll take a look at this soon.

**Comment by mattiasdrp** [comment]
Done in #542
